extension/package.json

```diff
@@ -554,8 +554,7 @@
       "title": "DVC",
       "properties": {
         "dvc.focusedProjects": {
-          "title": "%config.focusedProjects.title%",
```

Comment:
In https://github.com/iterative/vscode-dvc/pull/201 a title was added to a configuration property. For 3000+ PRs/issues we have persisted with the title field. It has no effect.

Reply:
probably even debated those titles.